### PR TITLE
Min/APPEALS-31621

### PR DESCRIPTION
**Pull request by minhazur9**
Resolves [APPEALS-31621](https://jira.devops.va.gov/browse/APPEALS-31621)

# Description
Added model for the transcription_transaction table

## Acceptance Criteria
- [ ] Code compiles correctly
Create a new Model called transcription_transaction_model
a "template" Model
- [ ] many transcription_transactions can have 1 transcript
- [ ] many transcription_transactions can have 1 appeal
- [ ] many transcription_transactions can belong to 1 docket
- [ ] belongs_to :transcription
- [ ] methods to upload retrieved files from Webex to S3 locations
- [ ] Audio files to transcript_audio
- [ ] Raw VTT files to transcript_raw
- [ ] method to update transcription file status
- [ ] method to download transcription files from S3 to local
Create a new Controller called transcription_transactions_controller
a "template" Controller methods of
- [ ] index, new
- [ ] update
- [ ] create

## Testing Plan
1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/APPEALS-38473) 

- [ ] For feature branches merging into master: Was this deployed to UAT?

### Test Coverage
Did you include any test coverage for your code? Check below:
- [ ] RSpec
- [ ] Jest
- [ ] Other

### CODE REVIEW

**Comment by ammitche2** [comment]
thank you!

File: app/models/hearings/transcription_file.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class TranscriptionFile < CaseflowRecord
+  include BelongsToPolymorphicAppealConcern
+  belongs_to_polymorphic_appeal :appeal
+  belongs_to :transcription
+  belongs_to :transcript
```
Comment (msteele96) [inline app/models/hearings/transcription_file.rb:7]: I don't see a Transcript object in Caseflow for this to belong to

**Review by msteele96 — CHANGES_REQUESTED** [review]
In addition to the other comment, could you pull in latest from feature? I'm hoping these commits will fix a couple of the strange rspec failures.

File: app/models/hearings/transcription_file.rb (hunk 1 above)
Comment (minhazur9) [reply app/models/hearings/transcription_file.rb:7]: The AC stated this model is associated with a transcript. Is there any plans for a model object for Transcript? Or was it just a mistake?

**Review by msteele96 — APPROVED** [review]

**Pull request merged**